packages/rspack/src/builtin-plugin/SplitChunksPlugin.ts

```diff
@@ -119,6 +124,10 @@ function toRawSplitChunksOptions(
 					test: getTest(test),
 					name: getName(name),
 					chunks: getChunks(chunks),
+					minSize: JsSplitChunkSizes.__to_binding(minSize) as any,
```

Comment:
Should here use group.minSize and group.maxSize?

Reply:
Thank you!

> Should here use group.minSize and group.maxSize?

OMG you are right! I think might be clinically retarded 💀